setup.py

```diff
@@ -21,6 +22,13 @@
 with open('requirements.txt') as f:
     install_requires = f.read().splitlines()
 
+def package_files(directory):
```

Comment:
Educated "3rd" best Stackoverflow answer, that acutally is the best answer ;-)

Reply:
?

Reply:
https://stackoverflow.com/questions/27664504/how-to-add-package-data-recursively-in-python-setup-py/27664856

Reply:
Isn't MANIFEST.in enough?

Reply:
and adding '*.v'  after '*.yaml'.

Reply:
Also consider that the answers are more than 2 years old. Quite a bit for setuptools.

Reply:
>Isn't MANIFEST.in enough?

As I see it will be part of package but will not be installed during installation.